week-3/todo-list/script.js

```diff
@@ -1,25 +1,68 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
-
-// These are the same todos that currently display in the HTML
-// You will want to remove the ones in the current HTML after you have created them using JavaScript
 let todos = [
   { task: "Wash the dishes", completed: false },
   { task: "Do the shopping", completed: false },
 ];
 
-populateTodoList(todos);
-
-// This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
 function addNewTodo(event) {
-  // The code below prevents the page from refreshing when we click the 'Add Todo' button.
-  event.preventDefault();
-  // Write your code here... and remember to reset the input field to be blank after creating a todo!
+
+  let list = document.getElementById("todo-list");
+  let listValue = list.value;
+ 
+  
+  let li = document.createElement("li")
```

Comment:
Well done @BakhatBegum it is very readable code.
 You can improve your code by using `const` instead of `let` for variables that don't need to be reassigned. variables like `list`, `listValue`, `li` etc